src/main/java/com/cloudbees/jenkins/support/impl/CustomLogs.java

```diff
@@ -95,6 +95,30 @@ public Iterable<LogRecord> getLogRecords() {
                     }
                 });
             }
+
+            // Add rotated log files
+            File[] rotatedLogFiles = customLogs.listFiles((dir, filename) -> filename.matches(name + "\\.log\\.\\d+"));
```

Comment:
BTW I find
```suggestion
            File[] rotatedLogFiles = customLogs.listFiles((dir, filename) -> filename.matches(name + "[.]log[.][0-9]+"));
```
a bit more legible (so I do not need to mentally parse out backslashes).

Reply:
updated to [.]log[.][